src/Model/Item.php

```diff
@@ -1328,23 +1284,15 @@ private static function tagDeliver($uid, $item_id)
 
 		$forum_mode = (($prvgroup) ? 2 : 1);
 
-		q("UPDATE `item` SET `wall` = 1, `origin` = 1, `forum_mode` = %d, `owner-name` = '%s', `owner-link` = '%s', `owner-avatar` = '%s',
-			`private` = %d, `allow_cid` = '%s', `allow_gid` = '%s', `deny_cid` = '%s', `deny_gid` = '%s'  WHERE `id` = %d",
-			intval($forum_mode),
-			dbesc($c[0]['name']),
-			dbesc($c[0]['url']),
-			dbesc($c[0]['thumb']),
-			intval($private),
-			dbesc($u[0]['allow_cid']),
-			dbesc($u[0]['allow_gid']),
-			dbesc($u[0]['deny_cid']),
-			dbesc($u[0]['deny_gid']),
-			intval($item_id)
-		);
+		$fields = ['wall' => true, 'origin' => true, 'forum_mode' => $forum_mode,
+			'owner-name' => $c[0]['name'], 'owner-link' => $c[0]['url'], 'owner-avatar' => $c[0]['thumb'],
+			'private' => $private, 'allow_cid' => $u[0]['allow_cid'], 'allow_gid' => $u[0]['allow_gid'],
```

Comment:
Can you please replace the query by a `dba::selectFirst` call in line 1206 : https://github.com/friendica/friendica/pull/4487/files#diff-043fd70296cbf595b3caed1b2dc79248R1206 and use a meaningful variable name instead of `$u`?